modules/react/layout/stories/Grid.stories.mdx

```diff
@@ -0,0 +1,94 @@
+
+<ExampleCodeBlock code={UIExample} />
+
+### Rows and Columns
+
+You can nest `Grid` components to build layouts with rows and columns. You can also nest `Grid`
+components within `<Grid.Item/>` components for even more possibilities.
+
+In the below example, we have a `Grid` container. It contains nine elements with one of them being
+grey. This is a `<Grid.Item/>` component and has access to child specific props, the `<Grid/>`
+component only has access to container props. This is to show what the `<Grid.Item/>` component
+can&nbsp;do.
+
+In the layout, the `<Grid.Item/>` component (grey element), has a `Row` and `Col` button. These
+buttons are interactive and if you click the `Row` button, it will move the `<Grid.Item/>` element
+between the rows and if you click the `Col` button, it will move it through the columns. These
+buttons are updating `gridRowStart`(`grid-row-start` css grid property) and
+`gridColumnStart`(`grid-column-start` css grid property). This is to showcase the child specific
+props for the `<Grid.Item/>` component.
+
+<ExampleCodeBlock code={GridLayoutInteractive} />
+
+**⚠️This is NOT best practice for accessibility reasons and we suggest against following this. This
+is merely to showcase the child specific props for the `<Grid.Item/>` component.⚠️**
+
+In the second example below, we have a `Grid` container. It contains two rows with one having seven
+elements and the other has two elements. This example is to show how a user can use the
+`<Grid.Item/>` component to wrap a `<Grid/>` component. This would allow a user to have access to
+the child props of `<Grid.Item/>` to place a layout where needed. The row with elements 3 - 7 is put
+before the 1 & 2 elements since that layout is wrapped by the `<Grid.Item/>` component.
+
+<ExampleCodeBlock code={GridLayout} />
+
+### Grid vs. Box
+
+`Grid` is built with `Box` and has access to all `BoxProps`. The main differences between `Box` and
+`Grid` are:
+
+- `Grid` is set to `display: grid` & `inline-grid` by default.
+- The `display` prop is limited to `grid` & `inline-grid` .
+- `Grid` has access to grid container properties.
+- `Grid.Item` has access to grid child properties.
+
+In short, use `Grid` when you need a grid container to build layouts with CSS Grid, and use `Box`
+when you need a simple container. It's also important to note that `Grid` has no real opinion about
+how to build layouts outside of the CSS Grid spec. Its unopinonated nature allows it to be composed
+into many different components.
+
+## Grid Props
+
+As previously mentioned, `Grid` is built on top of `Box` and has access to all its props, including
+[grid container properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout#the_grid_container)
+through style props as well. Here are the grid container props:
+
+<ArgsTable of={GridStyleProps} />
+
+### Grid Props (Exhaustive List)
+
+<ArgsTable of={Grid} />
```

Comment:
This is the same as the `GridStyleProps`, so we can remove it. 👍🏻 
```suggestion
```

Reply:
Updated